docs/src/proposals/stake-v2.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Stake Account Version 2
+---
+
+The current stake program is battle-tested, but time has revealed a few important
+issues:
+
+1. Redelegations are not possible
+
+Whales can't easily redelegate their positions, since they do not want to lose
+out on an epoch of rewards.
+
+2. Rewards calculations at epoch boundaries take too long
+
+Since all of the stake accounts simultaneously receive rewards at the start of an epoch,
+the network experiences a big load to process these accounts. This feature
+is extremely useful for users, since they don't need to do anything to claim
+rewards. On the flip side, it introduces too much load on validators.
+
+With a new stake account, we can address both of these issues.
```

Comment:
My gut says it'll be better to split (1) and (2) into two different proposals that can run concurrently, but for now I'm keeping an open mind!